libs/pilight/events/actions/label.lua

```diff
@@ -61,9 +83,19 @@ function M.check(parameters)
 
 		if nr3 < nr2 then
 			error("label actions are formatted as \"label DEVICE ... TO ... FOR ...\"");
+			return -1;
```

Comment:
There is no need for the `return -1` if we trigger the error.

Reply:
done